src/Files.App/Data/Models/ItemViewModel.cs

```diff
@@ -1193,6 +1193,29 @@ public async Task LoadExtendedItemProperties(ListedItem item, uint thumbnailSize
 									gp.InitializeExtendedGroupHeaderInfoAsync();
 								}));
 						}
+
+						if (item.IsGitItem &&
+							GitHelpers.IsRepositoryEx(item.ItemPath, out var repoPath) &&
+							!string.IsNullOrEmpty(repoPath))
+						{
+							cts.Token.ThrowIfCancellationRequested();
+							await SafetyExtensions.IgnoreExceptions(() =>
+							{
+								var repo = new LibGit2Sharp.Repository(repoPath);
+								GitItemModel gitItemModel = GitHelpers.GetGitInformationForItem(repo, item.ItemPath);
+
+								return dispatcherQueue.EnqueueOrInvokeAsync(() =>
+								{
+									var gitItem = item.AsGitItem;
+									gitItem.UnmergedGitStatusLabel = gitItemModel.StatusHumanized;
+									gitItem.GitLastCommitDate = gitItemModel.LastCommit?.Author.When;
+									gitItem.GitLastCommitMessage = gitItemModel.LastCommit?.MessageShort;
+									gitItem.GitLastCommitAuthor = gitItemModel.LastCommit?.Author.Name;
+									gitItem.GitLastCommitSha = gitItemModel.LastCommit?.Sha.Substring(0, 7);
```

Comment:
```suggestion
									gitItem.GitLastCommitSha = gitItemModel.LastCommit?.Sha.Substring(0, 7);

									repo.Dispose();
```
`LibGit2Sharp.Repository` is disposable and should be disposed of when no longer needed.